checkST2110.js

```diff
@@ -758,18 +786,31 @@ const test_20_72_4 = (sdp, params) => {
   return errors;
 };
 
-// ST 2110-20 Section 7.2 Test 5 - Check SSN is the required fixed value
+const ssnPermitted20 = ['ST2110-20:2017', 'ST2110-20:2022'];
+
+// ST 2110-20:2017 Section 7.2
+// ST 2110-20:2022 Section 7.2 Test 5 - Check SSN is the required fixed value
 const test_20_72_5 = (sdp, params) => {
   let [mtParams, errors] = extractMTParams(sdp, params);
   for (let stream of mtParams) {
     if (typeof stream.SSN !== 'undefined') {
```

Comment:
How about if SSN is not present, but ALPHA or ST2115LOGS3 are?

Reply:
Good spot. Will add extra check.

Reply:
...according to ST 2110-20:2017 Section 7.3, SSN is a required parameter. So will already fail `test_20_72_1`, so extra an check is redundant.